source/Posets/2ElementPoset.lagda

```diff
@@ -0,0 +1,53 @@
+Ian Ray, 9 August 2023
+
+Constructing the two element poset.
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K --exact-split #-}
+
+open import MLTT.Spartan
+open import MLTT.Two-Properties
+open import UF.FunExt
+open import UF.PropTrunc
+open import UF.Logic
+
+open import UF.Subsingletons
+open import UF.Subsingletons-FunExt
+
+module Posets.2ElementPoset
+  (pt : propositional-truncations-exist)
+  (fe : Fun-Ext)
+  where
```

Comment:
Our convention is to indent with single spaces instead of the usual Agda convention of double spaces. For things like these, you can check out `CONTRIBUTING.md`.

Reply:
Okay I think I fixed all of these and I'll check out the CONTRIBUTING.md going forward.